platform/broadcom/docker-ptf-brcm.dep

```diff
@@ -2,6 +2,7 @@
 DPATH       := $($(DOCKER_PTF_BRCM)_PATH)
 DEP_FILES   := $(SONIC_COMMON_FILES_LIST) platform/broadcom/docker-ptf-brcm.mk platform/broadcom/docker-ptf-brcm.dep
 DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+DEP_FILES   += $(SONIC_COMMON_DOCKER_FILES_LIST)
```

Comment:
i do not get this part. which file is actually needed by building docker-ptf-brcm?

Reply:
This adds support for adding files that are common across all the dockers. Right now, the ptf-brcm might not have common $SONIC_COMMON_DOCKER_FILES_LIST dependency, but from the framework perspective, files might get added later.